src/main/resources/build.properties

```diff
@@ -2,3 +2,8 @@
 APPLICATION.NAME=${pom.name}
 APPLICATION.VERSION=${pom.version}
 APPLICATION.BUILDTIME=${maven.build.timestamp}
+GIT.COMMIT.CHECKSUM=${git.commit.id}
+GIT.BRANCH=${git.branch}
+GIT.CURRENT.TAG=${git.tag}
+GIT.CLOSEST.TAG=${git.closest.tag.name}
+GIT.COMMIT.TIME=${git.commit.author.time}
```

Comment:
Could you add the 5 extra properties to the "Initializing service..." log in `ContextListener`?